README.md

```diff
@@ -16,32 +16,32 @@ development and testing purposes.
 
 ### Prerequisites
 
-You will need to have Node.js and npm installed on your system.
+You will need to have `Node.js` and `pnpm` installed on your system.
 
 ### Installing
 
 1. Clone the repository:
 
 ```bash
-git clone https://github.com/Frelya/react-site-editor/react-page-builder.git
+git clone https://github.com/Frelya/react-site-editor.git
 ```
 
 2. Navigate to the project directory:
 
 ```bash
-cd react-site-builder
+cd react-site-editor
 ```
 
 3. Install the dependencies:
 
 ```bash
-npm i
+pnpm i
 ```
 
 4. Start the development server:
 
 ```bash
-npm run dev
+pnpm dev
```

Comment:
There's something missing. Not?

Reply:
No, nothing is missing. What are you thinking about?

Reply:
Something like pnpm run dev

Reply:
The `run` option is not mandatory in `pnpm`, as in `yarn`